src/tests/work-item/work-item-list/page-objects/work-item-list.page.js

```diff
@@ -38,7 +38,9 @@ class WorkItemListPage {
      browser.get("http://localhost:8088/");
    }
  };
-
+ workItemByURLId (workItemId) {
```

Comment:
I like the addition of the function that enables us to access a workitem's detail page directly through the detailpage URL! In fact, I'd like to see us add many more tests that access the UI's element through their respective direct URLs. 

I did find a way to resolve the problem that you are having. The key is to first access the workitem's detail page, and then access the date on that detail page. For example:

```
var detailPage = page.clickWorkItemTitle(page.workItemByTitle("Title Text 3"), "id3");
    expect(detailPage.getCreatedtime()).toBe('3 days ago');
    detailPage.clickWorkItemDetailCloseButton();
```
or
```
var detailPage = page.clickWorkItemTitle(page.workItemByIndex(3), "id3");
    expect(detailPage.getCreatedtime()).toBe('3 days ago');
    detailPage.clickWorkItemDetailCloseButton();
```